app/tests/util/member_personas.py

```diff
@@ -80,6 +83,69 @@ async def bob_and_alice_public_did(
     alice_acapy_client: AcaPyClient,
     bob_acapy_client: AcaPyClient,
 ) -> BobAlicePublicDid:
+    tenant_admin_acapy = governance_acapy_client()
+
+    invite = await tenant_admin_acapy.connection.create_invitation(
+        alias="endorser",
+        auto_accept=True,
+        public=True,
+        body=InvitationCreateRequest(
+            handshake_protocols=["https://didcomm.org/didexchange/1.0"],
+            use_public_did=True,
+        ),
+    )
+
+    # create did
+    set_public_did_result = await create_public_did(tenant_admin_acapy)
+
+    bob_conn = await bob_acapy_client.connection.receive_invitation(
+        alias="endorser", auto_accept=True, body=invite.invitation
+    )
+    alice_conn = await alice_acapy_client.connection.receive_invitation(
+        alias="endorser", auto_accept=True, body=invite.invitation
+    )
+
+    bob_records = await bob_acapy_client.connection.get_connections()
+    alice_records = await alice_acapy_client.connection.get_connections()
+
+    # Get connection record for admin
+    admin_conn_records = await tenant_admin_acapy.connection.get_connections()
+
+    set_endorser_role_result_bob = (
+        await bob_acapy_client.endorse_transaction.set_endorser_role(
+            conn_id=bob_conn.connection_id, transaction_my_job="TRANSACTION_AUTHOR"
+        )
+    )
+    set_endorser_info_result_bob = (
+        await bob_acapy_client.endorse_transaction.set_endorser_info(
+            conn_id=bob_conn.connection_id,
+            endorser_did=set_public_did_result.did,
+            endorser_name="endorser",
+        )
+    )
+    set_endorser_role_result_alice = (
+        await alice_acapy_client.endorse_transaction.set_endorser_role(
+            conn_id=alice_conn.connection_id, transaction_my_job="TRANSACTION_AUTHOR"
+        )
+    )
+    set_endorser_info_result_alice = (
+        await alice_acapy_client.endorse_transaction.set_endorser_info(
+            conn_id=alice_conn.connection_id,
+            endorser_did=set_public_did_result.did,
+            endorser_name="endorser",
+        )
```

Comment:
Why are you setting up endorsement between alice and bob? Is this because of the bug present in ACA-Py 0.7.4?

Reply:
No this isn't needed I think. Sorry, leftover from experimenting